sp041_20191205T1757.cdl

```diff
@@ -332,4 +331,46 @@ data:
  PLATFORM_MODEL = "Spray" ;
```

Comment:
I'm not sure if this is producing a full example file. The mandatory terms under PLATFORM are 'PLATFORM_TYPE' and 'PLATFORM_MODEL'
For platform type you can reference - https://vocab.nerc.ac.uk/collection/L06/current/
For platform model you can reference - https://vocab.nerc.ac.uk/collection/B76/current/

spray has a couple of versions in the collection 
https://vocab.nerc.ac.uk/collection/B76/current/B7600028/
https://vocab.nerc.ac.uk/collection/B76/current/B7600027/

Reply:
Thank you for the careful reading! It's fixed.